plugin-ci/orb.yml

```diff
@@ -104,11 +104,21 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
```

Comment:
This could also live under `deploy-release-github`, but then another docker image would be needed. `circleci/golang:1.13-node` brings `jq` with it.